src/test/java/racinggame/car/CarTest.java

```diff
@@ -0,0 +1,29 @@
+package racinggame.car;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class CarTest {
+    @Test
+    void 주사위값이_4이상이면_자동차위치를_전진시킬_수_있다() {
+        Car car = Car.of();
+
+        car.changePosition(4);
+
+        assertThat(
+                car.getPosition()
+        ).isEqualTo(2);
```

Comment:
```suggestion
        assertThat(car.getPosition()).isEqualTo(2);
```
위와 같이 한 라인으로 구현해도 되지 않을까?